computer-fundamentals/basics.md

```diff
@@ -163,61 +150,57 @@ Consider everyday "smart" devices that may not look like computers but have feat
 <details><summary>More possible answers</summary>
 1. Smartwatches: They contain computer systems, including processors, memory, and storage, allowing them to run applications, track health data, receive notifications, and perform various functions. They have operating systems and can connect to the internet, enabling them to process data, execute tasks, and interact with other devices.
 2. Smart TVs: They are essentially computers integrated with televisions. They feature powerful processors, memory, and storage, running operating systems specifically designed for smart TV functionalities. They can connect to the internet, stream content from online services, run applications, and offer interactive features.
-</details>
+</details> -->
 
 > Note: A shortcut to find out if a system is a computer (in the strictest sense of the word) is to check for "Turing-completeness". This is a theoretical concept which measures how flexible and versatile a system is when it comes to solving problems, thus determining whether or not is has the capabilities of a general-purpose computer. Embedded devices are usually not Turing complete, while smartphones are. To learn more, read the first section of [this article](https://academy.binance.com/en/glossary/turing-complete).
 
-# Questions
+## Questions
```

Comment:
Questions will be moved to the article's front-matter per [this PR in Knewhub](https://github.com/knewplay/knewhub/pull/37).
I will let @hpyhacking comment on if this change should be part of this PR or a future one for this repository.

Reply:
That's fine, I'll make those changes here

Reply:
Instead of :

questions:
  - ["jacquard", "Why isn't the Jacquard loom considered to be the first computer?"]
  - ["mouse", "Does a computer absolutely need to have a keyboard and a mouse in order for it to be a computer?"]
...

Can we do :

questions:
  1. ["jacquard", "Why isn't the Jacquard loom considered to be the first computer?"]
  2. ["mouse", "Does a computer absolutely need to have a keyboard and a mouse in order for it to be a computer?"]

Reply:
The YAML syntax uses dashes (`-`) so we should stick with it. I can modify the parser on my end to display a numbered list as the end result, if we wish.